content/ember-4-0-released.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Ember 4.0 released
+authors:
+  - jen-weber
+date: 2021-11-22T00:00:00.000Z
+tags:
+  - releases
+  - '2021'
+  - version-4-x
+---
+
+
+Today, the Ember project is releasing version 4.0.0 of Ember.js, Ember Data, and
+Ember CLI. Ember it focuses
+the framework by removing long-deprecated APIs and support for legacy platforms.
+With these changes, Ember Octane becomes the norm!
+As planned, this release introduces no new features.
+
+This blog post will help you understand how to upgrade and navigate any breaking changes.
+
+Additionally, today the 4.1 beta cycle begins for these same projects.
+
+Finally, today we're also promoting Ember 3.28 to LTS ([3.28 release
+post](https://blog.emberjs.com/ember-3-28-released)).
+
+## Our major release strategy
+
+Why are there no new features in this release?
+This has been the tradition for all major Ember releases since v2.
+We believe that major versions should be rare, and that when they happen,
+they should not introduce new features. 
+
+This strategy aims to give Ember
+developers the time and space to upgrade dependencies and 
+navigate any breaking changes, without needing to
+learn lots of new things at the same time.
+In many libraries including Ember, a major release means that there are breaking changes.
+And while it would be exciting for major releases to be splashy and full of
+new features,
+we think it is more important to reduce risks for our developers and
+the users that depend on them.
+
+Our hope is that that library maintainers across the JavaScript ecosystem
+will follow this pattern, which makes it easier for developers to keep up
+with security patches and bugfixes in their project's dependencies.
+
+Our plans for Ember 4.0 were originally announced in July 2021 in 
+[The Road to Ember 4.0](https://blog.emberjs.com/the-road-to-ember-4-0/).
+
+## Trying Ember for the first time
+
+If you're interested in trying Ember for the
+first time today, get started by running:
+
+```bash
+npx ember-cli new my-project
+cd my-project
+npx ember serve # Then visit http://localhost:4200
```

Comment:
does `npx ember` point at ember-cli?

Reply:
Not exactly? This is invalid:

```
npx ember new my-test-app
npm ERR! could not determine executable to run
```

This _is_ valid.
```
npx ember-cli serve
```

Any thoughts on whether we should be using `npx ember-cli` for all command examples?

Reply:
IMO npx makes sense for installation, but traditional use of the 'ember' command should be the norm after project setup.